disnake/ext/commands/core.py

```diff
@@ -1850,6 +1853,168 @@ async def predicate(ctx: AnyContext) -> bool:
     return check(predicate)
 
 
+def app_check(predicate: AppCheck) -> Callable[[T], T]:
```

Comment:
These two new decorators are missing from `checks.rst` and currently don't appear in the docs c:

Reply:
52a198aa36ac476e040941e7ff9c7391f976d2f5